distro/src/avail/Avail.avail/Foundation.avail/Lexers.avail

```diff
@@ -38,22 +38,25 @@ Uses
 	"Characters",
 	"Control Structures",
 	"Early Conditionals",
+	"Early Functions",
 	"Exceptions",
 	"Format",
-	"Literals",
+	"Literals" = (-"…#.…#«f»?", -"…#.…#…", -"…#.…#e|E«+|-»!…#«f»?",…),
```

Comment:
These messages don't exist anymore, right? So the restriction should be removed.

Reply:
Correct, all three exist nowhere but on this (about to be removed) line.